tests/__tests__/ts-errors.spec.ts

```diff
@@ -9,9 +9,20 @@ describe('hello_world', () => {
     const stderr = result.stderr.toString();
 
     expect(result.status).toBe(1);
-    expect(stderr).toContain('Hello.ts:13:11');
+    expect(stderr).toContain('Hello.ts:18:11');
```

Comment:
This doesn't look right. If the source file hasn't changed, why should the line number change in the test?